src/components/ui/UiNetworkFigure.vue

```diff
@@ -1,20 +1,31 @@
 <script setup lang="ts">
+import informationIcon from '../../assets/icons/informationIcon.svg'
 const props = defineProps({
   figure: Number,
   description: String,
   unit: String,
+  icon: String,
+  moreInformation: String,
 })
 </script>
 <template>
-  <div class="w-32 h-24">
-    <div class="text-black font-bold font-poppins not-italic leading-none">
-      <span class="text-4xl px-1">{{ props.figure }}</span>
-      <span class="text-base px-1">{{ props.unit }}</span>
+  <div class="w-32 h-24 font-dm-sans">
+    <div class="flex flex-row items-center p-0 gap-2 w-32 h-9">
+      <div class="flex flex-row leading-8 font-bold gap-1 items-end flex-grow">
```

Comment:
where did you find the class `flex-grow` ?
https://tailwindcss.com/docs/flex-grow

BTW in figma it's `grow:0` not `grow:1`